mne/chpi.py

```diff
@@ -292,24 +311,51 @@ def _fit_chpi_pos(coil_dev_rrs, coil_head_rrs, x0):
     return x, 1. - result / denom
 
 
+def _fit_dev_head_trans(dev_pnts, head_pnts):
```

Comment:
This should be renamed to something like `_fit_coil_order_dev_head_trans` because it does both, and based on the currenty name I thought you were going to also use this for the continuous fitting of (Neuromag) data.

Incidentally, Neuromag sometimes gets it wrong, so we might want to eventually use a function like this in cases where the coil locations are way off -- we could sample the data at some intervals and have a confidence estimate about which coils are where. But this is definitely something for later, this is a good starting point for it.